tests/utils/tracer.py

```diff
@@ -30,9 +29,12 @@ def write(self, spans=None, services=None):
             self.traces += trace
 
         if services:
-            self.json_encoder.encode_services(services)
-            self.msgpack_encoder.encode_services(services)
-            self.services.update(services)
+            deprecation(
+                name='write(services)',
+                message='Writing services has been replaced with specifying the'
```

Comment:
Please enclosing multi-line string in parenthesis:

```
message=(
    'Writing ...'
    'and span_type'
    'sent.'
),
```

Also, this can probably be 2 lines instead of 3.

The reason for doing the parenthesis is so that there isn't any confusion around what is a parameter vs a continuation, and helps when you notice there are no commas at the end of lines, etc.

This also helps give you more room for each string since it'll start on the next line.